README.md

```diff
@@ -117,6 +117,25 @@ export dbConnection=...
 delete_db_scxa_marker_genes.sh
 ```
 
+## `exp_design` and `exp_design_column` tables
+
+### Load data
+Run `bin/load_exp_design.sh`. It requires the following environment variables:
+
+| Variable name         | Description                                                                                                                                                                                                                                  |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CONDENSED_SDRF_FILE` | Path of the condensed SDRF file of the experiment being loaded                                                                                                                                                                               |
+| `SDRF_FILE`           | Path of the SDRF file of the experiment being loaded                                                                                                                                                                                      |
```

Comment:
I checked the README above and the name of the environment variable for the condensed sdrf file is different for other tables, so I placed this suggestion here based on that.
```suggestion
| `CONDENSED_SDRF_TSV `           | Path of the SDRF file of the experiment being loaded                                                                                                                                                                                      |
```